<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LightMdEditor - A Markdown Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionAnchor_Link"/>
    <addaction name="actionImage"/>
    <addaction name="actionTable"/>
    <addaction name="actionCode_Block"/>
    <addaction name="actionBlockquote"/>
    <addaction name="actionFind"/>
    <addaction name="actionFind_Replace"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionContact"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <addaction name="actionFont"/>
    <addaction name="actionFocus_Mode"/>
    <addaction name="actionWordWrap"/>
    <addaction name="actionFull_Screen"/>
    <addaction name="actionTheme_Toggle"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionBlockquote"/>
   <addaction name="actionAnchor_Link"/>
   <addaction name="actionImage"/>
   <addaction name="actionTable"/>
   <addaction name="actionCode_Block"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/new.png</normaloff>:/images/theme_dark/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/edit.png</normaloff>:/images/theme_dark/edit.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/save.png</normaloff>:/images/theme_dark/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/cut.png</normaloff>:/images/theme_dark/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/copy.png</normaloff>:/images/theme_dark/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/paste.png</normaloff>:/images/theme_dark/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/bold.png</normaloff>:/images/theme_dark/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/italic.png</normaloff>:/images/theme_dark/italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAnchor_Link">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/link.png</normaloff>:/images/theme_dark/link.png</iconset>
   </property>
   <property name="text">
    <string>Anchor Link</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/image.png</normaloff>:/images/theme_dark/image.png</iconset>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionCode_Block">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/codeblock.png</normaloff>:/images/theme_dark/codeblock.png</iconset>
   </property>
   <property name="text">
    <string>Code Block</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+K</string>
   </property>
  </action>
  <action name="actionBlockquote">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/blockquote.png</normaloff>:/images/theme_dark/blockquote.png</iconset>
   </property>
   <property name="text">
    <string>Blockquote</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionContact">
   <property name="text">
    <string>Contact</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/font.png</normaloff>:/images/theme_dark/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/find.png</normaloff>:/images/theme_dark/find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_Replace">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/findreplace.png</normaloff>:/images/theme_dark/findreplace.png</iconset>
   </property>
   <property name="text">
    <string>Find &amp; Replace</string>
   </property>
  </action>
  <action name="actionWordWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>WordWrap</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="icon">
    <iconset resource="LightMdEditor.qrc">
     <normaloff>:/images/theme_dark/table.png</normaloff>:/images/theme_dark/table.png</iconset>
   </property>
   <property name="text">
    <string>Table</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionFocus_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Focus Mode</string>
   </property>
  </action>
  <action name="actionTheme_Toggle">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Theme Toggle</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LightMdEditor.qrc"/>
 </resources>
 <connections/>
</ui>
